meson.build

```diff
@@ -91,7 +91,7 @@ if get_option('with_wayland') != 'no'
     version : libva_version,
     required : get_option('with_wayland') == 'yes')
 
-  WITH_WAYLAND = wl_scanner.found() and libva_wayland_dep.found()
```

Comment:
Perhaps, a better fix is to move out the wl_scanner's find_program() out of the conditional, and add the wayland_client_dep.found() in the WITH_WAYLAND variable, keeping the wl_scanner.found()

Reply:
@ceyusa I'm not much familiar with these mechanics. Would you mind to open new PR with correct fix? I'll close this one then.

Reply:
@ceyusa Thx.